app/views/distributions/index.html.erb

```diff
@@ -41,6 +41,7 @@
                   <th>Source Inventory</th>
                   <th>Total items</th>
                   <th>Total value</th>
+                  <th>Status</th>
```

Comment:
Check how the Filters are done with the various models -- You don't have to add the filter-controls to the top like in the other cases, but you can use the `Filterable` concern to mediate this and give users an easy way to see the picked-up distributions.

Reply:
LMK if you need further info or have questions!